AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1337,6 +1337,9 @@ quotev.com#$#body #res_area ::selection { background-color: #3297FD!important; }
 !
 !--- JS ----
 !
+outdoornews.co.kr#%#//scriptlet('set-constant', 'document.body.oncontextmenu', 'null')
+outdoornews.co.kr#%#//scriptlet('set-constant', 'document.body.onselectstart', 'null')
+outdoornews.co.kr#%#//scriptlet('set-constant', 'document.body.ondragstart', 'null')
```

Comment:
What are these rules for?

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/90512/commits/ce1c4e69e929ab00ec350ca5cdaaeac3fe7800b8